### DISCUSSION TITLE
Send Notification only if there are errors

### DISCUSSION THREAD

**Discussion by khalidmaquilang** [spatie/laravel-health#144] [Q&A]
Is this possible?

**Comment by szhorvath** [comment]
I have the same problem. The table size check continuously sends an email that the table size is ok. Therefore you have to read every notification to check if something is not ok. Obviously, no one does that.

**Comment by danmatthews** [comment]
Also related to #168 - we're getting the same.

**Comment by freekmurze** [comment]
This has been fixed in the latest version of the package.
